pyproject.toml

```diff
@@ -39,6 +39,21 @@ docs = [
     "sphinx-lint==0.9.1",
 ]
 
+[tool.uv]
```

Comment:
Is it possible not to duplicate all the extra requirements here, but rather use ` --extra dev` / `--extra lint` / `--extra types` in the Makefile?

Reply:
It's possible, but the long-term goal is to remove those extras and just have them as dev dependencies. We can't currently set internal dependency groups as dev dependencies (see: https://github.com/astral-sh/uv/issues/7033) though.